consumer/consumererror/combine.go

```diff
@@ -21,18 +21,19 @@ import (
 
 type combined []error
 
-var _ error = (*combined)(nil)
-
 func (c combined) Error() string {
```

Comment:
will we want to deprecate Error as well or keep it?

Reply:
Error is a method on the object returned by the Combine. Once Combine is not used the Error is not used.

Reply:
Also a function on a private struct